.github/workflows/deploy-website.yml

```diff
@@ -132,9 +132,10 @@ jobs:
         uses: actions/checkout@master
 
       - name: Download Build Artifact
-        uses: actions/download-artifact@master
+        uses: actions/download-artifact@v2
         with:
           name: public
+          path: public
```

Comment:
With v1 of the `download-artifact` action, the artifact would automatically download into a directory with the same name. Since v2, you have to give it a `path`, otherwise the files in the artifact gets dumped in the root directory.